backend/app/routes/faq/index.js

```diff
@@ -0,0 +1,26 @@
+const {authMiddleware, generateJWT} = require('../../../helpers/middlewares/auth');
+const express = require ("express");
+const FAQ = require('../../models/faq');
+const router = express.Router();
+
+
+router.use('/', authMiddleware , async (req,res,next) => {
+    try {        
+        const newFAQ = new FAQ({
```

Comment:
you can directly use the req.body and need not create a new obj like this or use const FAQ = req.body
that should do the work

Reply:
Also this